src/unittest/test_compression.cpp

```diff
@@ -174,6 +176,42 @@ void TestCompression::testZlibLargeData()
 	}
 }
 
+void TestCompression::testZstdLargeData()
+{
+	infostream << "Test: Testing zstd wrappers with a large amount "
+		"of pseudorandom data" << std::endl;
+
+	u32 size = 500000;
```

Comment:
500.000 because zstd default buf size is 128k.

Reply:
No longer needed now since I changed the bufsize to 64, but it's still a good test, and takes only a few ms anyway.